### ISSUE TITLE
aria-selected should be mandatory if no aria-activedescendant is used

### ISSUE THREAD

**Issue by JAWS-test** [w3c/aria#1094]
ARIA specification about [aria-selected](https://w3c.github.io/aria/#aria-selected) says:

> Single-selection containers where the currently focused item is not selected. The selection normally follows the focus, and is managed by the user agent.

This sounds to me like it is not necessary to use `aria-selected` if only one element can be selected and the selected element is always identical with the focused element. I think this is wrong, because the focused element is only perceptible in form mode (focusing with tab key). If you read with the arrow keys, the focus is not on the element and therefore the selected element is not perceptible. 
The same applies to the quick navigation and the element overview (e.g. in JAWS for drop-down lists and tabs with C and INS+Ctrl+C).

>  If no DOM element in the widget is explicitly marked as selected, assistive technologies MAY convey implicit selection which follows the keyboard focus of the managed focus widget.

Confusing is also the "May", if no `aria-selected` is marked. This means that you can't rely on it and have to use `aria-selected`. However, this is not explicitly mentioned in the specification of the roles. For example, the [specification for Tab](https://w3c.github.io/aria/#tab) uses a "Should" for the same subject.

>  In the absence of an aria-selected attribute on the current tab, user agents SHOULD indicate to assistive technologies through the platform accessibility API that the currently focused tab is selected.

I suggest that the section be changed to always use aria-selected (Exception: use of aria-activedescendant).

Related: https://github.com/w3c/aria/issues/700
